File: c/ejercicios_todocode/comprobacion_de_valores/comprobacion_de_valores.c
/* Realizar un programa que permita cargar 15 numeros en un vector.
Una vez cargados, se necesita que el programa cuente e informe por pantalla cuantas veces se cargo el numero 3 */

#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main(void)
{
    srand(time(NULL));

    int numbers[15];
    int cont_three = 0;

    for (int i = 0; i < 15; i++)
    {
        numbers[i] = 1 + rand() % (15);
    }

    for (int i = 0; i < 15; i++)
    {
        if (numbers[i] == 3)
        {
            cont_three++;
        }
    }

    printf("El numeros 3 se repitio %i veces en el array.\n", cont_three);
    return 0;
}

File: python/ejercicios_todocode/comprobacion_de_valores/comprobacion_de_valores.py
# Realizar un programa que permita cargar 15 numeros en un vector.
# Una vez cargados, se necesita que el programa cuente e informe por pantalla cuantas veces se cargo el numero 3
import random

def main() -> int:
    numbers: list[int] = []
    cont_three: int = 0

    for i in range(15):
        numbers.append(random.randint(1, 15))

    for number in numbers:
        if number == 3:
            cont_three += 1
    
    print(f"El numeros 3 se cargo {cont_three} veces en el array.")
    print(numbers)
    return 0

if __name__ == "__main__":
    main()